### PR TITLE
readme updates - set up guides how to

### PR DESCRIPTION
**Pull request by smalho01**
## Describe your changes

Please include a summary of the changes and the related issue/task. Please also include relevant motivation and context. List any dependencies that are required for this change, including links to other pull requests/branches in other repositories if applicable.

## Issue ticket number and Jira link

Please include the Jira Ticket Number and Link for this issue/task.

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] Ensure the target / base branch for any feature PR is set to `dev` not main (the only exception to this is releases from `dev` and hotfix branches)

## Checklist for conducting a review
- [ ] Review the code changes and make sure they all make sense and are necessary. 
- [ ] Pull the PR branch locally and test by running through workflow and making sure everything works as it is supposed to. 

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.

### CODE REVIEW

File: DeveloperSetupGuide.md [hunk 1]
```diff
@@ -300,34 +300,45 @@ After rebuilding images and containers, start docker-sync normally
...
 8. After several seconds you should receive a response in the form of two **CDS cards**:
-    - **Drug Has REMS: Documentation Required.**
 9. Select **Patient Enrollment Form** on the returned CDS card with summary **Drug Has REMS: Documentation Required**.
 10. If you are asked for login credentials, use **alice** for username and **alice** for password.
 11. A webpage should open in a new tab, and after a few seconds, a questionnaire should appear.
-12. Fill out questionnaire and hit **Submit REMS Bundle**.
+12. Fill out the questionnaire and hit **Submit REMS Bundle**.
+    12a. Alternatively fill out only some of the questionnaire for an asynchronous workflow and hit **Save to EHR**.
+    12b. Visit the Patient Portal at http://localhost:3000/patient-portal and lay the role of the patient.
+    12c. Login to the Patient Portal, use **JonSnow** for the username and **jon** for the password.
```
Comment (smalho01) [inline DeveloperSetupGuide.md:321]: In Markdown, this just lists off as one big paragraph - need to figure out a way to make each sub step show up in a list

File: SimpleSetupGuide.md [hunk 2]
```diff
@@ -120,30 +120,41 @@ Your computer must have these minimum requirements:
...
+7. Click **Submit to REMS-Admin** at the bottom of the page, which demonstrates the case where an EHR has CDS Hooks implemented natively.
 8. After several seconds you should receive a response in the form of two **CDS cards**:
-    - **Drug Has REMS: Documentation Required.**
 9. Select **Patient Enrollment Form** on the returned CDS card with summary **Drug Has REMS: Documentation Required**.
 10. If you are asked for login credentials, use **alice** for username and **alice** for password.
 11. A webpage should open in a new tab, and after a few seconds, a questionnaire should appear.
-12. Fill out questionnaire and hit **Submit REMS Bundle**.
+12. Fill out the questionnaire and hit **Submit REMS Bundle**.
+    12a. Alternatively fill out only some of the questionnaire for an asynchronous workflow and hit **Save to EHR**.
+    12b. Visit the Patient Portal at http://localhost:3000/patient-portal and lay the role of the patient.
```
Comment (smalho01) [inline SimpleSetupGuide.md:136]: See previous comment

**Review by plarocque4 — APPROVED** [review]
Looks good

**Review by zacharyrobin — APPROVED** [review]
lgtm

**Pull request merged**